app/views/students/student_report.pdf.erb

```diff
@@ -89,8 +89,8 @@ ul, li {
       <div style="float: left; width:370px; padding:5px">
         <ul>
           <li><span style="font-weight:bold;"><%= format_date(service.date_started) %> -
-            <% if service.discontinued_services.count > 0%>
-              <%= "#{format_date(service.discontinued_services.last.try(:recorded_at))}" %>
+            <% if service.discontinued_at != nil%>
+              <%= "#{format_date(service.try(:recorded_at))}" %>
```

Comment:
@alexsoble when you test the student report pdf generating process, please make sure the `recorded_at` continues to show.